### PR TITLE
some a11y fixes

### PR DESCRIPTION
**Pull request by npiyush97**
## Hey, I just made a Pull Request!
#19339 

#### :heavy_check_mark: Checklist

- [ ] A changeset describing the change and affected packages. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#creating-changesets))
- [ ] Added or updated documentation
- [ ] Tests for new functionality and regression tests for bug fixes
- [ ] Screenshots attached (for UI changes)
- [X] All your commits have a `Signed-off-by` line in the message. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#developer-certificate-of-origin))

### CODE REVIEW

**Comment by npiyush97** [comment]
ping @emmaindal @adamdmharvey 
fixes also include for blog and plugin ribbon.

**Comment by npiyush97** [comment]
@awanlin changed the theme monokai to chalk lighthouse seems to be happy with this.

File: docs/overview/what-is-backstage.md [hunk 1]
```diff
@@ -40,7 +40,7 @@ Out of the box, Backstage includes:
 Backstage is a CNCF Incubation project after graduating from Sandbox. Read the announcement
 [here](https://backstage.io/blog/2022/03/16/backstage-turns-two#out-of-the-sandbox-and-into-incubation).
 
-<img src="https://backstage.io/img/cncf-white.svg" width="400" />
+<img src="https://backstage.io/img/cncf-white.svg" alt="cncf logo" width="400" />
```
Comment (awanlin) [inline docs/overview/what-is-backstage.md:43]:
```suggestion
<img src="https://backstage.io/img/cncf-white.svg" alt="CNCF logo" width="400" />
```

File: docs/overview/background.md [hunk 2]
```diff
@@ -35,4 +35,4 @@ Backstage was originally built by Spotify and then donated to the CNCF.
 Backstage is currently in the Incubation phase. Read the announcement
 [here](https://backstage.io/blog/2022/03/16/backstage-turns-two#out-of-the-sandbox-and-into-incubation).
 
-<img src="https://backstage.io/img/cncf-white.svg" width="400" />
+<img src="https://backstage.io/img/cncf-white.svg" alt='cncf logo' width="400" />
```
Comment (awanlin) [inline docs/overview/background.md:38]:
```suggestion
<img src="https://backstage.io/img/cncf-white.svg" alt="CNCF logo" width="400" />
```

File: .changeset/quick-months-clap.md [hunk 3]
```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-devtools': patch
+---
+
+Refactored code to improve accessibility by moving elements outside the `ul` tag and placing them appropriately.
```
Comment (awanlin) [inline .changeset/quick-months-clap.md:5]:
```suggestion
Refactored code to improve accessibility by moving elements outside the `ul` tag and placing them appropriately. Also adjusted theme to offer better contrast.
```

**Review by awanlin — CHANGES_REQUESTED** [review]
Thanks for the corrections, I've looked this over the best I can, the SCSS changes I'm unsure about.

Just a heads up as the owner of the DevTools plugin I got tagged on this. Ideally it would be great to have broken this up into two PRs, the issue you linked doesn't mention these other items and as you provided no description it was and is confusing. 👍

File: plugins/devtools/src/components/Content/ConfigContent/ConfigContent.tsx [hunk 4]
```diff
@@ -87,7 +87,7 @@ export const ConfigContent = () => {
           src={configInfo.config as object}
           name="config"
           enableClipboard={false}
-          theme={theme.palette.type === 'dark' ? 'monokai' : 'rjv-default'}
+          theme={theme.palette.type === 'dark' ? 'chalk' : 'rjv-default'}
```
[the remark this answers is not in the document]
Comment (awanlin) [reply plugins/devtools/src/components/Content/ConfigContent/ConfigContent.tsx:90]: Thinking about this more: does Lighthouse test the Dark theme? Or you tested the Dark theme and then changed it to `chalk` based on your testing? My assumption is that the Lighthouse test failure in https://github.com/backstage/backstage/issues/19339 is using the Light theme which would be `rjv-default`, right?

[the remark this answers is not in the document]
Comment (npiyush97) [reply plugins/devtools/src/components/Content/ConfigContent/ConfigContent.tsx:90]: as far as I know i use Lighthouse as well as Axetool they check contrast based on size and foreground color and background color in run test on dark mode only.
axetool is not happy with light theme contrast issue in light mode.
yes i changed to chalk and ### checked with both and they were happy with it 😂.

File: plugins/devtools/src/components/Content/InfoContent/InfoContent.tsx [hunk 5]
```diff
@@ -127,22 +128,16 @@ export const InfoContent = () => {
...
-          >
-            <ListItemAvatar>
-              <Avatar>
-                <FileCopyIcon />
-              </Avatar>
-            </ListItemAvatar>
-            <ListItemText primary="Copy Info to Clipboard" />
-          </ListItem>
         </List>
+        <Button
```
[the remark this answers is not in the document]
Comment (npiyush97) [reply plugins/devtools/src/components/Content/InfoContent/InfoContent.tsx:132]: old 
Screenshot from 2024-01-15 19-05-49
new
Screenshot from 2024-01-15 18-59-25
old
Screenshot from 2024-01-15 19-05-25
new
Screenshot from 2024-01-15 18-58-54

File: plugins/devtools/src/components/Content/ConfigContent/ConfigContent.tsx (hunk 4 above)
[the remark this answers is not in the document]
Comment (awanlin) [reply plugins/devtools/src/components/Content/ConfigContent/ConfigContent.tsx:90]: Ah, I assumed they tested with Light theme as the default, interesting that it's Dark. Thanks for following up 👍 

I do still think going the custom theme path would be better, gives the Adopter more options as to how they want to handle this

File: plugins/devtools/src/components/Content/InfoContent/InfoContent.tsx (hunk 5 above)
[the remark this answers is not in the document]
Comment (awanlin) [reply plugins/devtools/src/components/Content/InfoContent/InfoContent.tsx:132]: Thanks for the images. Seeing as the button get the boarder I think it's fine the way you have it.

For the theme, the "old" seems better to me, the text stands out better, what do your think looking at those images?

[the remark this answers is not in the document]
Comment (npiyush97) [reply plugins/devtools/src/components/Content/InfoContent/InfoContent.tsx:132]: so adding a divider and button is okay?
or add a divider and change back to an avatar?

[the remark this answers is not in the document]
Comment (awanlin) [reply plugins/devtools/src/components/Content/InfoContent/InfoContent.tsx:132]: The way you have it with a button is fine with me 👍 

I'm less sure on the theme change though

File: plugins/devtools/src/components/Content/ConfigContent/ConfigContent.tsx (hunk 4 above)
[the remark this answers is not in the document]
Comment (awanlin) [reply plugins/devtools/src/components/Content/ConfigContent/ConfigContent.tsx:90]: Looking at the images you shared the old looks better than the new so I'm not sure what to do 🤔 

I'm getting pretty busy with my work day. If I have some time I'll pull the changes down and run them locally

**Comment by npiyush97** [comment]
shifted to #22384

**Pull request closed without merging**